src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -102,9 +102,7 @@ private boolean testShouldWeAbort() {
     }
 
     if (serverGame != null && serverGame.getPlayerManager() != null) {
-      if (serverGame.getPlayerManager().isEmpty()) {
-        return true;
-      }
+      return serverGame.getPlayerManager().isEmpty();
```

Comment:
Not directly related either, but now the if statement can be merged with both return statements, by combining both boolean expressions with a logical AND